tests/tools/onert_train/src/nnfw_util.h

```diff
@@ -36,9 +36,9 @@ namespace onert_train
 uint64_t num_elems(const nnfw_tensorinfo *ti);
 uint64_t bufsize_for(const nnfw_tensorinfo *ti);
 
-std::string toString(NNFW_TRAIN_OPTIMIZER opt);
-std::string toString(NNFW_TRAIN_LOSS loss);
-std::string toString(NNFW_TRAIN_LOSS_REDUCTION loss_rdt);
+std::string to_string(NNFW_TRAIN_OPTIMIZER opt);
+std::string to_string(NNFW_TRAIN_LOSS loss);
+std::string to_string(NNFW_TRAIN_LOSS_REDUCTION loss_rdt);
 std::ostream &operator<<(std::ostream &os, const nnfw_train_info &info);
```

Comment:
Since above function `num_elems` and `bufsize_for` uses snake_cases,
Update this function also to have snake case!